home2.1/test_java.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
But what is the difference betwee test_java.sh and tests.sh?
I hope it can be easily handled with variable at the start of the file like
BINARY_TO_TEST=./task3 and BINARY_TO_TEST="java task3" which can be handled by input argument of the script